index.html

```diff
@@ -794,8 +794,8 @@ <h3>Address</h3>
                 <div class="info-box">
                   <i class="bi bi-telephone"></i>
                   <h3>Call Us</h3>
-                  <strong>Tanay Goel<br>(President):</strong><br><a href="tel: +917607073910">+917607073910</a><br><br>
-                  <strong>Arpit<br>(President):</strong><br><a href="tel: +919570099779">+919570099779</a><br>
+                  <strong>Suyash Bhagat<br>(President):</strong><br><a href="tel: +917835877316">+917835877316</a><br><br>
```

Comment:
> __[Secret Scan]__ It may be dangerous to commit the AWS account ID. ([view](https://sider.review/gh/repos/348849601/pulls/25#issue-6178531))
>
> <details>
>
> AWS account ID is a credential (12-digit number). Once this leaks, an attacker may identify essential information
> on your AWS account, such as roles, users, etc.
>
> See also:
> * https://docs.aws.amazon.com/general/latest/gr/acct-identifiers.html
> * https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html
> * https://rhinosecuritylabs.com/aws/assume-worst-aws-assume-role-enumeration/
>
> | Rule | Severity |
> | ---- | -------- |
> | `sider.secrets.aws.account_id` | `warning` |
>
> You can [close this issue](https://sider.review/gh/repos/348849601/pulls/25#issue-6178531) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).
>
> </details>

I closed this issue. Reason:
- When this is not an AWS account ID.